tensorrt_llm/bench/benchmark/low_latency.py

```diff
@@ -274,7 +299,16 @@ def latency_command(
     exec_settings["performance_options"]["cuda_graphs"] = True
     exec_settings["performance_options"]["multi_block_mode"] = True
 
-    exec_settings["extra_llm_api_options"] = params.get("extra_llm_api_options")
+    # Process recipe format if detected - extract llm_api_options only
+    # Priority: --recipe > --extra_llm_api_options
+    recipe_path = params.get("recipe", None)
+    extra_llm_api_options_path = params.get("extra_llm_api_options", None)
+    config_path = recipe_path if recipe_path else extra_llm_api_options_path
```

Comment:
I have concerns about this from a UX perspective. I don't think this is a matter of priority more so that they shouldn't be specified at the same time. See my comment above.